### DISCUSSION TITLE
packages for recent Fedora releases

### DISCUSSION THREAD

**Discussion by enpontus** [OpenOrienteering/mapper#2162] [Questions & Answers] [answered]
It seems as if OOMapper packages have come to Fedora 37 but not yet for F38, can this by any chance be fixed ? (I have some course setting tasks for the summer to tend to, and i did the upgrade to F38 a week ago (I have missed the availability of the recent releases up to 37 so did not bother to check at the time))

**Comment by enpontus** [comment]
I saw with joy and have used the recent releases, is it possible to have it built for Fedora 40 that is now available ? (i have not yet updated to that release due to OOmapper.

As always thanks for this software, and if possible thanks in advance for an update to f40 :D .

**Reply by dg0yt** [reply]
Thanks for asking, Fedora 40 packages are available now.

**Reply by enpontus** [reply]
Thanks a lot - i will give it a shot this weekend!

**Reply by enpontus** [reply]
perhaps not  a hurry (i will not upgrade just yet)  - but now Fedora 41 is out, so might be good to initiate builds for that, thanks

**Reply by enpontus** [reply]
is it possible to get Fedora 41 packages built - i am now planning an upgrade this week

**Reply by dg0yt** [reply]
> is it possible to get Fedora 41 packages built

Thanks for the reminder. I had to make some adjustments to the build process, but now Fedora 41 RPMs are rolling out.

**Comment by dg0yt** [comment, marked as answer]
RPMs for Fedora 41 are published now.

**Comment by enpontus** [comment]
Fedora 42 was released last week i think, can you add that build as well ?

**Reply by dg0yt** [reply]
Glad to see that there is interest. I enabled the build for the openorienteering-mapper-master package now, but there seems to be an OBS problem with the Fedora42 repository.

**Reply by dg0yt** [reply]
OBS seems to have fixed the issues. A successful build finished at Tue Apr 22 12:14:46 UTC 2025.

**Reply by enpontus** [reply]
:)

**Comment by enpontus** [comment]
And now F44 was released, jumped onto that early due to change of computer. So is it possible to build that ?

**Reply by dg0yt** [reply]
Builds are enabled now for F44.
